build-image/first-boot.bash

```diff
@@ -46,20 +46,20 @@ if is_pi; then
 fi
 
 echo -n "$(timestamp) [openHABian] Changing default username and password... "
-if [ -z ${username+x} ] || ! id $userdef &>/dev/null || id "$username" &>/dev/null; then
+if [ -z "${username:-openhabian}" ] || ! id $userdef &>/dev/null || id "$username" &>/dev/null; then
   echo "SKIPPED"
 else
   usermod -l "$username" $userdef
-  usermod -m -d "/home/$username" "$username"
-  groupmod -n "$username" $userdef
-  chpasswd <<< "$username:$userpw"
+  usermod -m -d "/home/$username $username"
+  groupmod -n "$username $userdef"
```

Comment:
Why have you concatenated these strings?

Reply:
because it worked for the previous line :-)
ok, "..." "..." now here as well